StringFormatterBench/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Text.Formatting;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Diagnostics.Windows;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+
+namespace StringFormatterBench
```

Comment:
I'd prefer to keep the same bracing style for all of the code in this project, for consistency.